### PR TITLE
Feature: Header & nav block

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: hedvig.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   clientPath: path.resolve(__dirname, 'build/assets/'), // Build asset path
   serverPath: path.resolve(__dirname, 'build/'), // Build asset path
   port: 8031, // The WDS port
-  developmentPublicPath: 'http://0.0.0.0:8031/', // Client public path during development, i.e. "http://0.0.0.0:8081/". Port must match the port directive
+  developmentPublicPath: 'http://10.10.150.74:8031/', // Client public path during development, i.e. "http://0.0.0.0:8081/". Port must match the port directive
```
Comment (palmfjord) [inline hedvig.config.js]: oops

File: src/server/page.tsx [hunk 2]
```diff
@@ -103,15 +101,28 @@ export const getPageMiddleware: Koa.Middleware = async (ctx) => {
   const routerContext: StaticRouterContext & { statusCode?: number } = {}
   const helmetContext = {}
 
-  const story = await getStoryblokResponseFromContext(ctx)
+  const [story, globalStory] = await Promise.all([
+    await getStoryblokResponseFromContext(ctx),
```
Comment (palmfjord) [inline src/server/page.tsx]: this shouldn't be await 🤨

File: src/blocks/HeaderBlock/index.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+)
+const Filler = styled('div')({
+  height: WRAPPER_HEIGHT,
+})
+const HeaderBackgroundFiller = styled('div')({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  right: 0,
+  zIndex: -1,
```
Comment (frapperino) [inline src/blocks/HeaderBlock/index.tsx]: wouldn't this impact the page content?

File: src/blocks/HeaderBlock/mobile.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+} from '../../components/blockHelpers'
+import { HEADER_VERTICAL_PADDING, TOGGLE_TRANSITION_TIME } from './index'
+
+export const TABLET_BP_DOWN = '@media (max-width: 800px)'
+export const TABLET_BP_UP = '@media (min-width: 801px)'
+
+export interface MobileVisibility {
+  isOpen: boolean
+  isClosing: boolean
+}
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/blocks/HeaderBlock/mobile.tsx]: good point! done ✅ the header also got a lot less messy!

**Comment by palmfjord** [comment]
@isaacSennerholt can i haz another review? :D

File: src/blocks/HeaderBlock/index.tsx [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+    '&:first-of-type': {
+      paddingTop: 0,
+    },
+  },
+})
+
+const LogoLink = styled('a')({
+  display: 'inline-flex',
+  paddingTop: 3, // fix to push down logo a little or it looks unbalanced
```
Comment (Iamsamwhoami) [inline src/blocks/HeaderBlock/index.tsx]: no comments 🙅‍♂️

File: src/components/icons/HedvigWordmark.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+})
+
+export const HedvigWordmark: React.FunctionComponent<{ height: number }> = ({
+  height,
+}) => (
+  <Svg
+    width={String(height * 3.2)}
+    height={String(height)}
+    viewBox="0 0 96 30"
+    version="1.1"
```
Comment (Iamsamwhoami) [inline src/components/icons/HedvigWordmark.tsx]: version is unnecessary :)

File: src/blocks/HeaderBlock/index.tsx (hunk 5 above)
Comment (palmfjord) [reply src/blocks/HeaderBlock/index.tsx]: I don't agree, i think we should _avoid_ comments but i don't think we should ban them completely. Sometimes they make sense. I would have to name this something like `PUSH__DOWN_LOGO_TO_AVOID_UNBALANCED_LOOK` which is just weird imo

Comment (palmfjord) [reply src/blocks/HeaderBlock/index.tsx]: (ignoring this after irl discussion :) )

**Pull request merged**